### ISSUE TITLE
Buttons take focus twice in Firefox

### ISSUE THREAD

**Issue by jakub-g** [ariatemplates/ariatemplates#1365]
http://snippets.ariatemplates.com/samples/github.com/ariatemplates/documentation-code/samples/widgets/button/style/

only in Firefox: the `@aria:Button` takes focus twice; first the `<button>` itself, then the last `<span>` inside of , without any good reason (it doesn't have `tabindex` specified)
